### PR TITLE
set minimum timestamp date for charts

### PR DESCRIPTION
**Pull request by Pantani**
# Headline
set minimum timestamp date for charts 

## Issue
`time_start` must be a valid ISO 8601 timestamp or unix time value

closes https://github.com/trustwallet/blockatlas/issues/708

### CODE REVIEW

**Comment by vikmeup** [comment]
Where is this magic number come from?

**Comment by Pantani** [comment]
> Where is this magic number come from?

`time_start` must be a valid ISO 8601 timestamp or unix time value. If you try to use the CMC api for charts, this is the minimum value accepted is `1000000000`

Wrong:
`https://web-api.coinmarketcap.com/v1/cryptocurrency/quotes/historical?convert=USD&format=chart_crypto_details&id=1027&interval=1d&time_end=1579117918&time_start=999999999`

Right:
`https://web-api.coinmarketcap.com/v1/cryptocurrency/quotes/historical?convert=USD&format=chart_crypto_details&id=1027&interval=1d&time_end=1579117918&time_start=1000000000`

File: marketdata/charts.go [hunk 1]
```diff
@@ -28,6 +32,9 @@ func InitCharts() *Charts {
 
 func (c *Charts) GetChartData(coin uint, token string, currency string, timeStart int64) (blockatlas.ChartData, error) {
 	chartsData := blockatlas.ChartData{}
+	if timeStart < minUnixData {
+		timeStart = minUnixData
```
Comment (vikmeup) [inline marketdata/charts.go]: `max(timeStart, minUnixData)` like this

File: marketdata/charts.go [hunk 2]
```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/trustwallet/blockatlas/pkg/errors"
 )
 
+const (
+	minUnixData = 1000000000
```
Comment (vikmeup) [inline marketdata/charts.go]: rename to `minUnixTime `

File: marketdata/charts.go (hunk 1 above)
Comment (Pantani) [reply marketdata/charts.go]: Do you mean to create a method to verify the max value?

```
func max(timeStart, min int64) int64 {
	if timeStart < min {
		return min
	}
	return timeStart
}
```

Comment (vikmeup) [reply marketdata/charts.go]: There is a function for `Max()`: https://github.com/trustwallet/blockatlas/blob/master/pkg/numbers/number.go#L10

**Pull request merged**